src/utill.rs

```
@@ -176,7 +175,9 @@ pub fn setup_taker_logger(filter: LevelFilter, is_stdout: bool) {
 
         let config = config.build(root_logger).unwrap();
 
-        log4rs::init_config(config).unwrap();
+        if let Err(e) = log4rs::init_config(config) {
+            eprintln!("Failed to initialize logger: {}", e);
+        }
```

any reason why we are suppressing the error?

---

I was getting a my local branch specific error,so i tried to handle a particular error case,thanks for asking this,i have revert it back to `unwrap()`. You can review it.